<aura:component implements="force:lightningQuickActionWithoutHeader,force:hasRecordId" controller="helloAssoManager">
    <aura:attribute name="hasErrors" type="Boolean" description="Indicate whether there were failures or not" default="false" />
    <aura:attribute name="recordId" type="String"/>
    <aura:attribute name="errorFromDedup" type="String"/>
    <aura:attribute name="error" type="String"/>
     <aura:attribute name="simpleRecord" type="Object"/>
    <aura:attribute name="record" type="Object"/>
    <!-- <aura:handler name="init" value="{!this}" action="{!c.init}" />  -->
    <force:recordData aura:id="doublon"
        recordId="{!v.recordId}"
        layoutType="FULL"
        mode="VIEW"
        targetRecord="{!v.record}"
        targetFields="{!v.simpleRecord}"
        targetError="{!v.error}"
        recordUpdated="{! c.nodedup }"/>
    
    <aura:if isTrue="{!v.hasErrors}">
        <div class="userCreateError">
            <ui:message title="Error" severity="error" closable="true">
                Une erreur est survenue<br/>
                {!v.errorFromDedup}
            </ui:message>
        </div>
      <!--  <aura:set attribute="else">
            <lightning:button variant="brand" label="Re-Processer (j'ai fais le nÃ©cessaire)" onclick="{! c.dedup }" /><br/>
            <lightning:button variant="brand" label="Re-processer (ignorer le doublon)" onclick="{! c.nodedup }" /><br/>
        </aura:set>-->
    </aura:if>
    
    

</aura:component>